src/util/irep_serialization.cpp

```diff
@@ -95,8 +96,7 @@ void irep_serializationt::read_irep(
 
   if(in.get()!=0)
   {
-    std::cerr << "irep not terminated\n";
-    throw 0;
+    throw deserialization_exceptiont("irep not terminated");
```

Comment:
Where is this caught?

Reply:
@peterschrammel This is now sorted with #2996